bin/content-style

```diff
@@ -1,30 +1,32 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
 
-$LOAD_PATH.unshift File.join(File.dirname(__FILE__), *%w(.. lib))
-
 require 'content_style'
 require 'YAML'
 
-command = ARGV[0]
-
-dir_path = File.expand_path(command)
-config_path = File.read(File.join(Dir.pwd, '/config/content-style.yml'))
+config_path = File.read(File.join(Dir.pwd, 'config', 'content-style.yml'))
 config_file = YAML.load(config_path)
 config = config_file.fetch('ContentStyle')
 filetype = config.fetch('filetype')
-@linter = ContentStyle::Linter.new(config)
-files = Dir.glob(dir_path + '/**/*.' + filetype)
 
-files.each do |file|
+command = ARGV[0] || Dir.pwd
+dir_path = File.expand_path(command)
+files = Dir.glob(File.join(dir_path, '**', "*.#{filetype}"))
+
+linter = ContentStyle::Linter.new(config)
+
+violations = files.flat_map do |file|
```

Comment:
This will collect all violations across all files, so that we can easily know if there was a violation for any file.